servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -1783,6 +1806,166 @@ private void setStateFromEntry(SetStateEntry entry) throws FileDoesNotExistExcep
   }
 
   /**
+   * Sets the acl of a file or directory.
+   *
+   * @param path to be set acl on
+   * @param options acl option to be set
+   * @return true if set successfully, false otherwise
+   * @throws TachyonException
+   */
+  public boolean setAcl(TachyonURI path, SetAclOptions options) throws TachyonException {
```

Comment:
Question: so here in this method, we could set either owner/group/permission once at time, not possible to set multiple features together, right?
First, why we want this constraint? I don't see any technical blockers here
Second, if that is the case, please javadoc this requirement.